### ISSUE TITLE
Error in f1dataR::load_quali(2015, 16)

### ISSUE THREAD

**Issue by pbulsink** [SCasanova/f1dataR#50]
Error in `f1dataR::load_quali(2015, 16)`
```
! Can't subset columns that don't exist.
✖ Column `Q3` doesn't exist.
```

**Comment by pbulsink** [comment]
Note: Q3 was cancelled for weather in this race, see https://en.wikipedia.org/wiki/2015_United_States_Grand_Prix. We should fill the column with NA values.

**Comment by SCasanova** [comment]
Closed by #51
